test/test_cli_clever.py

```diff
@@ -28,14 +28,14 @@ def test_help(self):
     out, err, code = self.run_clever('-h')
     self.assertEqual(code, 0)
 
-  def test_api_key(self):
-    # Check for error when key is not provided
-    out, err, code = self.run_clever('district all')
-    self.assertEqual(code, 2)
-    # Check for no error when key is provided via -k
-    out, err, code = self.run_clever('district all -k DEMO_KEY')
-    self.assertEqual(code, 0)
-    # Check for no error when key is provided via CLEVER_API_KEY
-    env = {'CLEVER_API_KEY':'DEMO_KEY'}
-    out, err, code = self.run_clever('district all', env)
-    self.assertEqual(code, 0)
+  # def test_api_key(self):
```

Comment:
I have a task to clean up OR remove this CLI since it ONLY supports API keys right now. If we keep it, this test would be replaced with an API token test. Hence leaving it in for now